pkg/kubelet/cm/dra/plugin/client.go

```diff
@@ -141,7 +141,8 @@ func (r *draPluginClient) NodeUnprepareResource(
 	claimName string,
 	cdiDevices []string,
 ) (*drapbv1.NodeUnprepareResourceResponse, error) {
-	klog.V(4).InfoS(
+	logger := klog.FromContext(ctx)
+	logger.V(4).Info(
```

Comment:
Looks like the [commit](https://github.com/kubernetes/kubernetes/pull/119012/files#diff-0b0c13558c483aa8d0584bdce871e43cd3b2f1b521208a10dd6515561b9acd15) has migrated the file to contextual logging, please fetch the lastest master branch and rebase.